client/src/glossary/glossary.js

```diff
@@ -166,6 +166,11 @@ const Glossary_ = ({ active_key, items_by_letter }) => (
   </div>
 );
 
+function focusBTT() {
```

Comment:
just a minor thing, i'm not too sure using BTT would exactly be the clearest immediately to new readers of the code, so maybe it would be better to have the full name written? At the same time I feel like the meaning of `focusBTT` can be a bit ambiguous possibly? I was thinking `back_to_top_focus_item` might be a bit clearer but it's kinda long to say haha. Feel free to use something else that is clearer and shorter.

As a side note, in this case, we want to keep this as a defined function (non-anonymous) because in dev/prod, the function isn't gonna be run at every single change on the page, thus we don't want this function to be recreated on every render cycle.

if it wasn't like this, the alternative would be to creating a class method.